modules/screen-order-checkout/index.js

```diff
@@ -1,26 +1,399 @@
+    padding: 10,
+    paddingLeft: 20,
+    marginVertical: 10,
+    width: "100%"
+  },
+  history: {
+    paddingHorizontal: 20
+  },
+  historyDate: {
+    fontSize: 16,
+    marginVertical: 10,
+    color: "#9B9B9B",
+    marginLeft: 20
+  },
+  detailsCard: {
+    marginVertical: 10,
+    borderRadius: 10,
+    shadowColor: "rgba(0, 0, 0, 0.5)",
+    elevation: 5,
+    backgroundColor: "#fff",
+    paddingHorizontal: 20,
+    paddingVertical: 10
+  },
+  detailsHeading: {
+    fontSize: 16,
+    color: "grey",
+    marginBottom: 20
+  },
+  pricing: {
+    marginHorizontal: 20,
+    flex: 1,
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    lineHeight: 20
+  },
+  pricingText: {
+    fontSize: 14,
+    lineHeight: 20,
+    marginVertical: 1,
+    color: "#111112"
+  },
+  summaryText: {
+    marginVertical: 1,
+    fontSize: 14,
+    fontWeight: "bold",
+    color: "#111112",
+    lineHeight: 20
+  },
+  chartContainer: {
+    paddingHorizontal: 20
+  },
+  cardContent: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    marginVertical: 3
+  },
+  chartText: {
+    fontSize: 14,
+    fontWeight: "bold",
+    color: "#1E2022"
+  },
+  cartImage: {
+    width: 31,
+    height: 23
+  }
+});
+export default BillingPreferencesScreen;
+
+const TabView = ({ tabTitles, selected }) => {
+  return (
+    <View style={tabViewStyles.paletteContainer}>
+      {tabTitles.map((title, index) => (
+        <View
+          style={
+            index === selected
+              ? tabViewStyles.selected
+              : tabViewStyles.unSelected
+          }
+          key={index}
+        >
+          <Text>{title}</Text>
+        </View>
+      ))}
+    </View>
+  );
+};
+
+const tabViewStyles = StyleSheet.create({
+  paletteContainer: {
+    flex: 1,
+    height: 48,
+    backgroundColor: "#F1F1F1",
+    flexDirection: "row",
+    alignItems: "center",
+    borderRadius: 10,
+    padding: 6,
+    marginVertical: 10
+  },
+  selected: {
+    borderRadius: 10,
+    flex: 1,
+    backgroundColor: "#fff",
+    height: "100%",
+    justifyContent: "center",
+    alignItems: "center",
+    shadowColor: "gray",
+    elevation: 10
+  },
+  unSelected: {
+    flex: 1,
+    height: "100%",
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundColor: "#F1F1F1",
+    borderRadius: 10
+  }
+});
+
+const OrderCard = () => {
+  return (
+    <View style={orderStyles.container}>
+      <View style={orderStyles.order}>
+        <View style={orderStyles.image}>
+          <Image source={require("./assets/edit.png")} />
+        </View>
+        <View style={orderStyles.description}>
+          <Text>Order name</Text>
+          <View style={orderStyles.bottomComponent}>
+            <View style={orderStyles.quantity}>
+              <Text>-</Text>
+              <Text
+                style={{
+                  fontWeight: "bold"
+                }}
```

Comment:
Move inline styling to the stylesheet.

Reply:
write code for the card section as well instead of image